Makefile

```
@@ -0,0 +1,45 @@
+REPO_URL := $(shell git config --get remote.origin.url)
+REPO_SLUG := $(shell echo "$(REPO_URL)" | sed -E 's#(git@|https://)github.com[:/ ]##; s/\\.git$$//')
+REPO_NAME := $(shell basename "$(REPO_SLUG)")
+REPO_OWNER := $(shell dirname  "$(REPO_SLUG)")
+REPO_SLUG := $(or $(REPO_SLUG),$(REPOSITORY))
+REPO_OWNER := $(or $(REPO_OWNER),$(OWNER))
+REPO_NAME := $(or $(REPO_NAME),$(NAME))
+DEFAULT_BRANCH := $(shell git rev-parse --abbrev-ref origin/HEAD 2>/dev/null | sed 's#^origin/##')
+DEFAULT_BRANCH ?= main
+DEMO_URL := https://$(REPO_OWNER).github.io/$(REPO_NAME)/
+
+.PHONY: info install dev build test lint docs serve clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)